src/validateOptions.js

```diff
@@ -6,13 +6,14 @@ const {
 } = require('./constants')
 
 const clientMethods = ['getUrl', 'config']
-const booleanFlags = ['assets', 'raw', 'compress', 'drafts']
+const booleanFlags = ['assets', 'raw', 'compress', 'drafts', 'useInconsistentCursor']
 const numberFlags = ['maxAssetRetries', 'maxRetries', 'assetConcurrency', 'readTimeout']
 const exportDefaults = {
   compress: true,
   drafts: true,
   assets: true,
   raw: false,
+  useInconsistentCursor: false,
```

Comment:
Open for using another name for this option

Reply:
`mode?: ExportMode`, where the mode is either `stream` or `cursor`? We can simply document that using cursors can result in inconsistent data.

Booleans aren't so future-proof, in that you get in trouble if you ever need more than two modes.

Reply:
Me gusta